src/binaryutils.hpp

```diff
@@ -20,8 +20,14 @@
 
 #include "tensorinfo.hpp"
 
+namespace inference {
+class ModelInferRequest_InferInputTensor;
```

Comment:
The type is `inference::ModelInferRequest::InferInputTensor;`
therefore it should be
```
namespace inference {
  namespace ModelInferRequest {
    class InferInputTensor;
  }
```